src/components/Filters/FilterChecklist.vue

```diff
@@ -48,9 +48,11 @@
           <LicenseIcons v-if="filterType === 'licenses'" :license="item.code" />
           {{ itemLabel(item) }}
         </label>
+        
```

Comment:
Please, remove the spaces on the empty line (you can also run `npm run format` to remove them automatically)